CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+**Table of Contents**
+- [Getting Started](#getting-started) (1st draft)
+- [Philosophy](#philosophy) (TBD)
+- [Reporting an Issue](#reporting-an-issue) (TBD)
+- [Pull Requests](#pull-requests) (1st draft)
+	-  _Should this be renamed to "fixing an issue"?_
+	-  Link to GitHub guide for PRs
+	-  What issues are good to start with? (Eg. `feature`, `bug`, etc. )
```

Comment:
"help welcome" but more specifiically "beginner friendly"... although somewhere we need to disclaim what we mean by "beginner".

Reply:
To go down that Rabbit hole... do we mean beginner to _the project_ or to _coding_?

Reply:
Ha, perhaps the whole tags are named poorly.  I mean that for someone who:

- knows Javascript
- has a working familiarity with Regex (you won't get too far without this)
- perhaps has some rough familiarity with the idea of parsing text in general
- willingness to read our grammar documentation and look at other grammars

If I think I could possibly fix something in 5-30 minutes I'd consider it potentially beginner friendly.  (ie it's perhaps a simpler rule change that doesn't get into the nuance of our grammars or complex things).  Or there are some issues only involving HTML/CSS/themes/docs... I would tag those beginner friendly.  I guess I meant "to the project"?

I'm open to renaming the tags or coming up with a different classification system if it would be helpful.

I slap most things that don't affect the core parser with "help wanted"... things that I'd like to do myself or where I think someone might be more of a hindrance than help I just leave untagged (and someone could always join those discussions).